26_map-filter-reduce/26_map-filter-reduce.cljs

```diff
@@ -0,0 +1,30 @@
+;; -=-=--=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-==--=
+;; An Introduction to ClojureScript for Light Table users
+;; -==-=--=-=-=-=-=-=-=-=-==-=--=-=-==--==-=-=-=-=--===--==-
+
+(ns lt-cljs-tutorial
+  (:require [clojure.string :as string]))
```

Comment:
This require isn't used. I'd remove it.
(ns lt-cljs-tutorial
+  ~~(:require [clojure.string :as string])~~)